src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -24,6 +24,15 @@
 
 namespace System.Net.Http
 {
+    #region Enums
+    internal enum CookieUsePolicy
```

Comment:
Seems like this should be a nested enum at the end of the file along with the rest of the types there.